FinalTask/FinalTask/Model/Month.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace FinalTask.Model
+{
+    public enum Month
```

Comment:
As alternative you may use something like this in your class which create user instances - https://stackoverflow.com/questions/899565/is-there-a-predefined-enumeration-for-month-in-the-net-library instead of creation your own enum structure